apis/v1alpha2/gatewayclass_types.go

```diff
@@ -47,12 +60,19 @@ type GatewayClass struct {
 	Status GatewayClassStatus `json:"status,omitempty"`
 }
 
+const (
+	// GatewayClassFinalizerGatewaysExist MUST be added as a finalizer to the
```

Comment:
literally "must" is already "commanded or requested",  strong.
dont think the uppercase is still needed, people might feel a little bit scaring. 
 ```suggestion
	// GatewayClassFinalizerGatewaysExist must be added as a finalizer to the
```

Reply:
I'm not really sure how to distinguish between `MUST` and `must` for our godocs, but I think historically we've been using all caps to emphasize/match RFC terminology.

Reply:
https://www.rfc-editor.org/rfc/rfc2119